wapitiCore/attack/mod_methods.py

```diff
@@ -39,6 +39,48 @@ class ModuleMethods(Attack):
     do_post = True
     excluded_path = set()
 
+    async def test_compare_methods(self, request: Request, methods: set):
+        response_dict = {}
+        # posting a GET request to compare
+        # with the uncommon methods
+        try:
+            log_verbose(f"[+] {request}")
+            get_response = await self.crawler.async_send(request)
+        except RequestError:
```

Comment:
Is it relevant to make a request without verifying that GET is an option ? Let's save a request by verifying that GET exists before calling this method.